apps/database/supabase/migrations/20260615131025_add_course_module_quiz_submissions.sql

```
@@ -0,0 +1,65 @@
+grant truncate on table "public"."course_module_quiz_submissions" to "authenticated";
+grant update on table "public"."course_module_quiz_submissions" to "authenticated";
+
+grant delete on table "public"."course_module_quiz_submissions" to "service_role";
+grant insert on table "public"."course_module_quiz_submissions" to "service_role";
+grant references on table "public"."course_module_quiz_submissions" to "service_role";
+grant select on table "public"."course_module_quiz_submissions" to "service_role";
+grant trigger on table "public"."course_module_quiz_submissions" to "service_role";
+grant truncate on table "public"."course_module_quiz_submissions" to "service_role";
+grant update on table "public"."course_module_quiz_submissions" to "service_role";
+
+create policy "Enable users to view their own submissions only"
```

SQLFluff (4.2.1)

[error] Do not use special characters in identifiers.

(RF05)